bin/node/cli/src/chain_spec.rs

```diff
@@ -369,6 +370,7 @@ pub fn testnet_genesis(
 			compute: Default::default(),
 			storage: Default::default(),
 			trash_data_count: Default::default(),
+			..Default::default()
```

Comment:
Nit.

`glutton: Default::default(),`

Similar to other entries that have all the fields initialized as defaults

Reply:
done